ext/hal/nordic/CMakeLists.txt

```diff
@@ -11,3 +11,9 @@ if(CONFIG_HAS_NORDIC_DRIVERS)
 
   add_subdirectory(drivers)
 endif()
+
+if(CONFIG_HAS_NRFX)
```

Comment:
Is there a reason this change and the kconfig changes aren't in the glue logic commit instead?

Reply:
Actually, there isn't.
I'll move these changes to the next commit, as it seems to be a better place for them, indeed.